authors/tests/test_pagination.py

```diff
@@ -0,0 +1,40 @@
+# python and django imports
+import json
+from rest_framework.views import status
+from django.urls import reverse
+
+# local imports
+from .base_test import TestBase
+
+
+class TestPagination(TestBase):
+    """This class has tests for pagination setting."""
+
+    def get_token(self):
```

Comment:
This method could serve better if it was defined in the `TestBase` class, don't you agree?

Reply:
Yes